endpoints/echo/Dockerfile.container-engine

```diff
@@ -0,0 +1,12 @@
+FROM debian:jessie
```

Comment:
Why not just use `gcr.io/google_appengine/python` - it's designed to work on GKE as well.